### PR TITLE
docs: update supported versions section

### PR DESCRIPTION
**Pull request by DeerMichel**
#### Description of Change
6.0.x release preparation (https://github.com/electron/electron/projects/20#card-24035642).

cc @sofianguy 

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] relevant documentation is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)

#### Release Notes

Notes: none

### CODE REVIEW

File: docs/tutorial/support.md [hunk 1]
```diff
@@ -48,9 +48,9 @@ basis for some older supported lines. All contested decisions around release
 line backports will be resolved by the [Releases Working Group](https://github.com/electron/governance/tree/master/wg-releases) as an agenda item at their weekly meeting the week the backport PR is raised.
 
 ### Currently supported versions
+- 6.x
```
Comment (codebytere) [inline docs/tutorial/support.md]: nit: let's put `6.x.y`, `5.x.y` etc. for consistency with referenced in the rest of this file.

**Comment by sofianguy** [comment]
Marking WIP for now. This should be merged on *Monday, July 29*

**Comment by codebytere** [comment]
@DeerMichel there's trailing whitespace somewhere causing lint failures

**Comment by DeerMichel** [comment]
I really appreciate that it doesn't fix it itself 💁

**Pull request merged**